assets/js/modules/tagmanager/setup.js

```diff
@@ -486,37 +532,48 @@ class TagmanagerSetup extends Component {
 						onEnhancedChange={ this.handleAccountChange }
 						outlined
 					>
-						{ accounts.map( ( account ) => {
-							return (
-								<Option
-									key={ account.accountId /* Capitalization rule exception: `accountId` is a property of an API returned value. */ }
-									value={ account.accountId /* Capitalization rule exception: `accountId` is a property of an API returned value. */ }>
-									{ account.name }
-								</Option>
-							);
-						} ) }
+						{ []
+							.concat( accounts )
```

Comment:
As mentioned, we should include a "Select an account..." option as first one (placeholder option).

Reply:
Added back

Reply:
Removed again 😄